examples/producer/main.go

```diff
@@ -99,7 +99,11 @@ RETRY:
 					},
 					},
 				}
-				data.SetVersion("v1") //nolint:errcheck
+
+				if err := data.SetVersion("v1"); err != nil {
```

Comment:
we should ignore this  error , so that events are delivered
_=data.SetVersion("v1")

Reply:
if the error did occur and we ignore it, the event would be failed to sent later due to marshalling error. 
This is test code.

Reply:
ok, this is sample .